engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -344,6 +403,137 @@ public PDDocument createPDFFromText(Reader text, int pageLimit)
+                {
+                    try
+                    {
+                        font = PDType0Font.load(doc, mapping.getFont().getOriginalData());
+                    }
+                    catch (Exception e)
+                    {
+                        logger.error("Error loading mapping font: " + e.getMessage());
+                    }
+                }
+                else
+                {
+                    String nameWithExtension = name + ".ttf";
+
+                    FontFileFinder fontFileFinder = new FontFileFinder();
+                    List<URI> uris = fontFileFinder.find();
+
+                    for (URI uri : uris)
+                    {
+
+                        if (uri.getPath().contains(nameWithExtension))
+                        {
+                            InputStream fontIS = null;
+                            try
+                            {
+                                fontIS = new FileInputStream(new File(uri));
+                                if (null != fontIS)
+                                {
+                                    PDDocument documentMock = new PDDocument();
+                                    font = PDType0Font.load(documentMock, fontIS);
+                                    break;
+                                }
+                            }
+                            catch (IOException ioe)
+                            {
+                                String msg = "Error loading font " + name + " : " + ioe.getMessage();
+                                logger.error(msg, ioe);
+                            }
+                            finally
+                            {
+                                if (fontIS != null)
+                                {
+                                    try
+                                    {
+                                        fontIS.close();
+                                    }
+                                    catch (Exception e)
+                                    {
+                                        logger.error("Error closing font inputstream: " + e.getMessage());
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            if (font == null)
+            {
+                if (!name.equals(defaultFont))
+                {
+                    font = getFont(doc, defaultFont);
+                }
+                else
+                {
+                    font = getFont();
+                }
+            }
+
+            return font;
+        }
+
+        public String getFontName()
+        {
+            return this.fontName;
+        }
+
+        public String getDefaultFont()
+        {
+            if (defaultFont == null || defaultFont.isBlank())
+            {
+                return PDType1Font.HELVETICA.getName();

```

Comment:
In the config it looks like the default is NotoSans-Regular, but here it's Helvetica. Is there a reason for two different default fonts? Do we need to ship Helvetica in the docker image (as we're doing with NotoSans)?

Reply:
I have created a new `DEFAULT_FONT` constant containing `NotoSans-Regular` value so it can be used instead of `Helvetica`, even it's not expected to fallback in this validation, we can keep the consistency in the default font.